runtime-testsuite/resources/org/antlr/v4/test/runtime/templates/LexerExec/CharSet.stg

```diff
@@ -1,21 +1,17 @@
 TestType() ::= "Lexer"
 
-Options ::= [
-	"Ignore": "Currently unable to generate the required CRLF input sequence."
-]
-
 Grammar ::= [
 	"L": {<grammar("L")>}
 ]
 
-Input() ::= "34<\u000D><\n> 34"
```

Comment:
Even if true, if carriage returns are working then there is no need to change this test. I prefer to address this change (and the other files with the same exclusion) separately.